ConfigurationSystem/Client/Utilities.py

```diff
@@ -137,6 +151,12 @@ def getGridCEs(vo, bdiiInfo=None, ceBlackList=None, hostURL=None, glue2=False):
 
 def getSiteUpdates(vo, bdiiInfo=None, log=None):
   """ Get all the necessary updates for the already defined sites and CEs
+
+      :param str vo: VO name
+      :param dict bdiiInfo: information from DBII
+      :param log object: logger
```

Comment:
```suggestion
      :param log: logger
```
Removing the "type", otherwise the log and object would need to be switched

Reply:
fixed